src/Management/src/EndpointBase/HeapDump/HeapDumper.cs

```diff
@@ -0,0 +1,56 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the Apache 2.0 License.
+// See the LICENSE file in the project root for more information.
+
+using Microsoft.Diagnostics.NETCore.Client;
+using Microsoft.Extensions.Logging;
+using System;
+using System.Diagnostics;
+
+namespace Steeltoe.Management.Endpoint.HeapDump
+{
+    public class HeapDumper : IHeapDumper
```

Comment:
This is `LinuxHeapDumper` but renamed and observing the new HeapDumpType option

Reply:
Note that there is no support for changing dump type via parameter, only app configuration